go/service/user.go

```diff
@@ -129,3 +130,18 @@ func (h *UserHandler) LoadPublicKeys(_ context.Context, arg keybase1.LoadPublicK
 	}
 	return publicKeys, nil
 }
+
+func (h *UserHandler) LoadAllPublicKeysUnverified(_ context.Context,
+	arg keybase1.LoadAllPublicKeysUnverifiedArg) (keys []keybase1.PublicKey, err error) {
+
+	body := &jsonw.Wrapper{}
+	u, err := libkb.LoadUserFromServer(h.G(), arg.Uid, body)
+	if err != nil {
+		return
+	}
+	var publicKeys []keybase1.PublicKey
+	if u.GetKeyFamily() != nil {
```

Comment:
It appears to be all of them:

       if @public_keys.all?
          res.public_keys or= {}
          # This includes old versions of PGP keys that may be revoked or
          # superseded.
          res.public_keys.all_bundles = (key.bundle for key in @public_keys.all)